<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>339</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QPushButton" name="btn_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="btn_0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="btn_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="btn_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="btn_ch_up">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>ch_up</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="btn_ch_down">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>ch_down</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="btn_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="btn_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="btn_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
            </property>
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btn_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="btn_vol_down">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>vol_down</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="btn_vol_up">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>vol_up</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="screen">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>information</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_vol_up</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>up_volume()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_vol_down</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>down_volume()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ch_up</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>up_ch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ch_down</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>down_ch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_9()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>up_volume()</slot>
  <slot>down_volume()</slot>
  <slot>up_ch()</slot>
  <slot>down_ch()</slot>
  <slot>set_1()</slot>
  <slot>set_2()</slot>
  <slot>set_3()</slot>
  <slot>set_4()</slot>
  <slot>set_5()</slot>
  <slot>set_6()</slot>
  <slot>set_7()</slot>
  <slot>set_8()</slot>
  <slot>set_9()</slot>
  <slot>set_0()</slot>
 </slots>
</ui>
